src/file-handler.js

```diff
@@ -58,7 +79,7 @@ const mergeConfigFiles = async ({
     await writeFile(
       vscodeFileUri,
       Buffer.from(
-        JSON.stringify(deepMerge(vscodeFileContents, merged), null, 2)
+        JSON.stringify({ ...vscodeFileContents, ...merged }, null, 2)
```

Comment:
Ran into a bug caused by this while testing things out. We definitely do not want to do a complete deep merge here because doing so will result in the final result (e.g. `settings.json`) not being idempotent.

For example, if I have `settings.local.json` and `settings.shared.json` defined, respectively, as below:

```json
{
  "foo": [1, 2]
}
```

```json
{
  "foo": [3, 4]
}
```

The first evaluation (VS Code launch, user save, etc.) will produce the correct/expected `settings.json`:

```json
{
  "foo": [1, 2, 3, 4]
}
```

However, if we were to make any type of change to either the local or the shared file (e.g. add `"bar": true`, add a space or newline, etc.) then we end up with the _wrong_ result because of that deep merge doubling the result: 


```json
{
  "foo": [1, 2, 3, 4, 1, 2, 3, 4],
  "bar": true
}
```

This then compounds upon each evaluations with the arrays doubling every time.

The original spread-based merge may not suffice for hypothetical future additions/features, e.g. #109, but our [current posture as stated in the docs](https://github.com/swellaby/vscode-workspace-config-plus#limitations) is essentially "don't modify these files by hand". As such all we really needed to worry about is not completely nuking some property added to the native VS Code file that wasn't specified in the shared/local files, especially because some other extensions will automatically add content to those native files that we didn't want to wipe